jsribosome.go

```diff
@@ -551,6 +551,37 @@ func NewJSRibosome(h *Holochain, zome *Zome) (n Ribosome, err error) {
 		result, _ := jsr.vm.ToValue(entryHash.String())
 		return result
 	})
+
+	//===========================================================================
+	err = jsr.vm.Set("getBridges", func(call otto.FunctionCall) otto.Value {
+		a := &ActionGetBridges{}
+		args := a.Args()
+		err := jsProcessArgs(&jsr, args, call.ArgumentList)
+		if err != nil {
+			return mkOttoErr(&jsr, err.Error())
+		}
+		r, err := a.Do(h)
+		if err != nil {
+			return mkOttoErr(&jsr, err.Error())
+		}
+		var code string
+		for i, b := range r.([]Bridge) {
+			if i > 0 {
+				code += ","
+			}
+			if b.Side == BridgeTo {
+				code += fmt.Sprintf(`{Side:%d,Token:"%s"}`, b.Side, b.Token)
+			} else {
+				code += fmt.Sprintf(`{Side:%d,ToApp:"%s"}`, b.Side, b.ToApp.String())
+			}
+		}
+		code = "[" + code + "]"
+		object, _ := jsr.vm.Object(code)
+		result, _ := jsr.vm.ToValue(object)
+
+		return result
+	})
+
```

Comment:
Just thinking, this looks like pretty straightforward boiler plate code for making actions accessible in the Ribosome. Actions do have a Name() function that is not used here.. couldn't we come up with generalised code that iterates over all Action types and creates bindings in the script engine? so that this code here wouldn't be needed? Just as a general remark.

Reply:
Yah, How about you add a tech debt ticket about this?